recipe-box-frontend/src/main/frontend/src/components/RecipeView/RecipeViewGrid.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import RecipeDetails from './RecipeDetails';
+import RecipeImageDetails from './RecipeImageDetails';
+import { Recipe } from './types';
+import "./RecipeViewGrid.css";
+
+const RecipeViewGrid: React.FC = () => {
```

Comment:
You can set what's in here inside RecipeView, if it's not used in any other page